trade-finance-manager-api/src/v1/controllers/deal.submit.controller.js

```diff
@@ -107,7 +107,7 @@ const submitDealAfterUkefIds = async (dealId, dealType, checker, auditDetails) =
     }
 
     // Update the deal with all the above modifications
-    const tfmDeal = api.updateDeal({ dealId, dealUpdate, auditDetails });
+    const tfmDeal = await api.updateDeal({ dealId, dealUpdate, auditDetails });
```

Comment:
This is left out on purpose to avoid sending the update deal stage.

Reply:
I'm also confused @abhi-markan 

All the api calls in this file are an `await`, quite surprised this was missing

If needs to _not_ await (why? - bizarre) can we add a comment about this please.

Reply:
Apologies both, I misunderstood this with deal status controller. Good check @MarRobSoftwire `await` should have been there inline with LOC:183.

I tested this and it appears due to an unexpected response from `addPartyUrn` which causes a snowball effect. I would have expected this to fail in last 6 months but above must have developed a race condition.

Reply:
@MarRobSoftwire when you get a minute can you please set the base to `2.4.0` so we can regress this on `staging` next so we can get this out on `prod`.